glasgow2024/templates/nominate/bits/nomination_instructions.html

```diff
@@ -8,6 +8,39 @@
 * profile -- The Member Profile
 * request.user -- The logged in user
 {% endcomment %}
-<h1>Glasgow in 2024 asks that you Nominate Good Things!</h1>
-<p>The Hugo Awards recognize Good Things™ in SF/F. Suggest to us some!</p>
-<p>雨果奖表役科幻/奇幻领域中的佳作。推荐一些给我们吧！</p>
+<h1>Nominating Ballot for the 2024 Hugo Awards, Lodestar Award for Best Young Adult Book, and Astounding Award for Best New Writer
+  
+Deadline</h1>
+<p>All ballots must be received by Saturday, 9 March 2024, 16:00 Greenwich Mean Time (GMT) (UTC+0)</p>
+
+<h1>How to Nominate</h1>
+<p>Enter your nominations in the text fields of the appropriate categories below. This form does not autosave. You will need to click the 'Save this Category' buttons at the bottom of each category, or 'Save All' at the bottom of the page.</p>
+
+<p>When you save your ballot below, you are signing your ballot as NAME.</p>
```

Comment:
Understandable that you'd be confused. Here's the change that will get you the result you want (otherwise this looks great):
```suggestion
<p>When you save your ballot below, you are signing your ballot as {{ request.user.convention_profile.preferred_name }}.</p>
```